spec/component_guide/component_example_spec.rb

```diff
@@ -26,7 +26,7 @@
 
   it 'passes params in examples to component example' do
     visit '/component-guide/test-component-with-params'
-    expect(body).to include('A test component that takes a required parameter')
+    expect(body).to include('A test component that takes a required')
```

Comment:
Not sure what this means? It made more sense before the removal of the word 'parameter'.

Reply:
Fair enough, I went with a different solution.

Reply:
And added some more context to the component in https://github.com/alphagov/govuk_publishing_components/pull/325/commits/d831255e5acb0e11932e02bee3e535389c1ed46a